addons/controllers/clusterbootstrap_controller.go

```diff
@@ -349,11 +366,11 @@ func (r *ClusterBootstrapReconciler) cloneSecretsAndProviders(cluster *clusterap
 	var createdSecrets []*corev1.Secret
 
 	packages := append([]*runtanzuv1alpha3.ClusterBootstrapPackage{
-		bootstrap.Spec.CNI,
 		bootstrap.Spec.CPI,
 		bootstrap.Spec.CSI,
 		bootstrap.Spec.Kapp,
-	}, bootstrap.Spec.AdditionalPackages...)
+	}, bootstrap.Spec.CNIs...)
+	packages = append(packages, bootstrap.Spec.AdditionalPackages...)
```

Comment:
This line could be moved up into the previous append

Reply:
No, cause now both ```CNIs``` and ```AdditionalPackages``` are slices, it doesn't let me to do that.